search-ui/src/composables/document-access-request-factory.ts

```diff
@@ -45,6 +52,18 @@ export const useDocumentAccessRequest = () => {
         documentAccessRequest._loading = false
     }
 
+    const getAccessRequestById = async (documentAccessRequestId: number) => {
```

Comment:
It doesn't look like this is being used by anything? Am I mistaken or can this be removed?

Reply:
Nice catch, I missed using it instead of direct call, did update this now.